Sources/Autoloader.php

```diff
@@ -40,7 +40,7 @@
 		$sourcedir = __DIR__;
 
 	// Do any third-party scripts want in on the fun?
-	if (class_exists('SMF\\Config', false) && $hook_value !== (Config::$modSettings['integrate_autoload'] ?? ''))
+	if (defined('SMF') && !defined('SMF_INSTALLING')  && class_exists('SMF\\Config', false) && $hook_value !== (Config::$modSettings['integrate_autoload'] ?? ''))
```

Comment:
I don't think any scenario exists in which the `SMF` constant wouldn't be defined by the time we got here. Am I missing something?